### PR TITLE
Build database

### PR DESCRIPTION
**Pull request by lenacha**
I made my database schema with sequelize and also implement seeding scripts.

### CODE REVIEW

File: db/generate.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+        bookingsByRoom[`${book.roomId}`] = [];
+      }
+      bookingsByRoom[`${book.roomId}`].push(book);
+    }
+  }
+}
+
+
+generateRandomRooms(1);
+// console.log(rooms);
```
Comment (weitsesun) [inline db/generate.js]: you may consider deleting debugging functions

Comment (lenacha) [reply db/generate.js]: okay! I will fix it! Thanks 👍

File: db/dataGenerator.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+const Sequelize = require('sequelize');
+
+const sequelize = new Sequelize('booking', 'yerin', 'yerin', {
```
Comment (erjena) [inline db/dataGenerator.js]: git ignore configs for your DB

Comment (lenacha) [reply db/dataGenerator.js]: Okay, I will change my password of database and add a config file for db. Thanks!

**Pull request merged**